### ISSUE TITLE
Home directory items inconsistent from L01 to L02

### ISSUE THREAD

**Issue by mjones01** [swcarpentry/shell-novice#289]
Caveat: I am going through the lesson on my own, not as part of a workshop.  I acknowledge this isn't how the lessons are designed to be used.  If this issue doesn't cause problems in workshops then please disregard and close the issue.  

In the second topic of The UNIX Shell lesson series, Files and Directories  (http://swcarpentry.github.io/shell-novice/01-filedir.html), Nelle's home directory (/Users/nelle/) contains subdirectories ~/Desktop/data-shell/  that contains other sub-directories and folders.  In topic 3, Creating Things (http://swcarpentry.github.io/shell-novice/02-create.html), the folders/files are in the home directory (/Users/nelle/) instead of in Desktop/data-shell.   I found this confusing when going from one lesson to the next.  However, it is arguably a minor issue.  

To make consistent, in L03 should start with /Users/nelle/Desktop/data-shell instead of just the home directory (/Users/nelle/Desktop/data-shell).

**Comment by shwina** [comment]
Thanks @mjones01 - nice catch! Here's the directory structure (I have this placed in my Desktop):

~~~
/home/shwina/Desktop

└── data-shell
    ├── creatures
    ├── data
    │   ├── elements
    │   └── pdb
    ├── Desktop
    ├── molecules
    ├── north-pacific-gyre
    │   └── 2012-07-03
    └── writing
        ├── data
        ├── old
        ├── thesis
        └── tools
            └── old
~~~

**Comment by ChristinaLK** [comment]
Once the above PR is merged, maybe open/create a new issue for removing the empty Desktop dir from data-shell?  That's a bigger project.

**Comment by mjones01** [comment]
This Issue appears to have been fixed with PR #314.  Close the issue?

**Comment by shwina** [comment]
Yes please. Thanks!
